package.json

```diff
@@ -1,20 +1,24 @@
 {
-    "name": "go-frontend",
-    "version": "7.0.0",
+    "name": "go-web-app",
+    "version": "1.0.0",
     "type": "module",
     "scripts": {
+        "initialize:type": "mkdir -p generated/ && touch generated/types.ts",
+        "prestart": "yarn initialize:type",
+        "prebuild": "yarn initialize:type",
+        "pretypecheck": "yarn initialize:type",
+        "prelint": "yarn initialize:type",
         "start": "vite",
-        "dev": "vite",
         "build": "vite build",
-        "preview": "vite preview",
-        "check-unused": "unimported",
-        "test": "vitest",
-        "coverage": "vitest run --coverage",
+        "generate:type": "dotenv -- cross-var openapi-typescript \"%APP_API_ENDPOINT%api-docs.yaml\" -o generated/types.ts --alphabetize",
         "typecheck": "tsc",
-        "lint": "eslint src",
-        "css-lint": "stylelint \"./src/**/*.css\"",
-        "deploy-surge": "branch=$(git rev-parse --symbolic-full-name --abbrev-ref HEAD); branch=$(echo $branch  | tr ./ -); cp build/index.html build/200.html; surge -p build/ -d https://ifrc-go-$branch.surge.sh",
-        "teardown-surge": "branch=$(git rev-parse --symbolic-full-name --abbrev-ref HEAD); branch=$(echo $branch  | tr ./ -); surge teardown https://ifrc-go-$branch.surge.sh"
+        "lint:js": "eslint src",
+        "lint:css": "stylelint \"./src/**/*.css\"",
+        "lint:unused": "unimported",
```

Comment:
Can we add `lint` command which will do `lint:js`, `lint:css`, `lint:unused` in one go?